cost-analyzer/templates/kubecost-metrics-service-monitor-template.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.kubecostMetrics }}
+{{- if .Values.kubecostMetrics.exporter }}
+{{- if .Values.kubecostMetrics.exporter.enabled }}
+{{- if .Values.kubecostMetrics.exporter.serviceMonitor }}
+{{- if .Values.kubecostMetrics.exporter.serviceMonitor.enabled }}
```

Comment:
:( Helm please

Reply:
it needs a `?` accessor so you can just write:
```
{{- if .Values.kubecostMetrics?.exporter?.serviceMonitor?.enabled }}
```